src/lib/components/progressbar/ProgressBar.component.js

```diff
@@ -0,0 +1,235 @@
+//@flow
+import React, { useEffect, useState } from 'react';
+import styled, { css } from "styled-components";
+
+export type ProgressBarProps = {
+  total: number,
+  filled: number,
+  size?: Size,
+  color: string,
+  measure: string,
+  percentage?: boolean,
+  capacity?: boolean,
+  label?: boolean
+};
+
+export const ProgressBarContainer = styled.div`
+  display: block;
+  background: #d9d9d9;
+
+  ${props => {
+    switch (props.size) {
+      case "smaller":
+        return css`
+          border-radius: 12px;
+          width: 200px;
```

Comment:
IMO, you do not need set the width, let it take the width of its parent (i.e Container).